contribs/gmf/src/directives/drawfeature.js

```diff
@@ -327,9 +367,38 @@ gmf.DrawfeatureController.prototype.handleMapClick_ = function(evt) {
     return ret;
   }.bind(this));
 
-  this.selectedFeature = feature ? feature : null;
+  feature = feature ? feature : null;
+
+  this.menu_.close();
+  if (feature) {
+    this.menu_.setPosition(coordinate);
+  }
+
+  this.selectedFeature = feature;
   this.scope_.$apply();
 };
 
 
+/**
+ * @param {ngeo.MenuEvent} evt Event.
+ * @private
+ */
+gmf.DrawfeatureController.prototype.handleMenuActionClick_ = function(evt) {
+  var action = evt.action;
+
+  switch (action) {
+    case gmf.DrawfeatureController.MenuActionType.DELETE:
+      goog.asserts.assert(
+          this.selectedFeature, 'Selected feature should be truthy');
+      this.features.remove(this.selectedFeature);
+      this.scope_.$apply();
+      break;
+    default:
+      // FIXME
```

Comment:
Nothing.  We don't have the 'move' and 'rotate' interactions enabled, yet.  The line below will disappear once we support those.